src/domain/session/room/timeline/tiles/SimpleTile.js

```diff
@@ -21,6 +21,7 @@ import {SendStatus} from "../../../../../matrix/room/sending/PendingEvent.js";
 export class SimpleTile extends ViewModel {
     constructor(options) {
         super(options);
+        this._tileCreator = options.tileCreator;
```

Comment:
Didn't we discuss passing `_tileCreator` as a parameter to `updateEntry`? If that doesn't work, could you remind me what the problem was?

Reply:
I also use `tileCreator` in` BaseMessageTile.replyTile`